DataFormats/PatCandidates/interface/PackedCandidate.h

```diff
@@ -661,6 +661,15 @@ namespace pat {
       setTrackProperties(tk, tk.covariance(), quality, covarianceVersion);
     }
 
+    void setTrackPropertiesLite(unsigned int covSchema,
+                                unsigned int covarianceVersion,
+                                unsigned int nHits,
+                                unsigned int nPixelHits) {
+      covarianceVersion_ = covarianceVersion;
+      covarianceSchema_ = covSchema;
+      packedHits_ = (nHits & trackPixelHitsMask) | (nPixelHits << trackStripHitsShift);
```

Comment:
this does not decode to the original input
```suggestion
      packedHits_ = (nPixelHits & trackPixelHitsMask) | ( ((nHits-nPixelHits) & trackStripHitsMask) << trackStripHitsShift);
```
apparently works